<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAnnotationModulePropertyDialog</class>
 <widget class="QDialog" name="qSlicerAnnotationModulePropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>464</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modify Annotation Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="5" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Name: AnnotationText (vtkMRMLAnnotationTextNode1)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="typeIcon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qSlicerAnnotationModule.qrc">:/Icons/AnnotationText.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="visibleInvisibleButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Toggle visibility</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerAnnotationModule.qrc">
         <normaloff>:/Icons/AnnotationVisibility.png</normaloff>
         <normalon>:/Icons/AnnotationInvisible.png</normalon>:/Icons/AnnotationVisibility.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lockUnlockButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Toggle un-/locked</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerAnnotationModule.qrc">
         <normaloff>:/Icons/AnnotationLock.png</normaloff>:/Icons/AnnotationLock.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" rowspan="3" colspan="2">
        <widget class="QTextEdit" name="annotationTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ctkColorPickerButton" name="textUnselectedColorPickerButton">
         <property name="text">
          <string>Unselected Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ctkColorPickerButton" name="textSelectedColorPickerButton">
         <property name="text">
          <string>Selected Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="textScaleLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Text Scale:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textOpacityLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkSliderWidget" name="textScaleSliderSpinBoxWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" stdset="0">
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ctkSliderWidget" name="textOpacitySliderSpinBoxWidget" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Measurement value:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="measurementLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Points</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="3" colspan="2">
        <widget class="QTableWidget" name="pointsTableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>457</width>
           <height>0</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>4</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ctkColorPickerButton" name="pointUnselectedColorPickerButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Unselected Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="pointGlyphScaleLabel">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="opacityLabel">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="ambientLabel">
           <property name="text">
            <string>Ambient:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="diffuseLabel">
           <property name="text">
            <string>Diffuse:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="specularLabel">
           <property name="text">
            <string>Specular:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ctkSliderWidget" name="pointOpacitySliderSpinBoxWidget" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="ctkSliderWidget" name="pointAmbientSliderSpinBoxWidget" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="ctkSliderWidget" name="pointDiffuseSliderSpinBoxWidget" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="ctkSliderWidget" name="pointSpecularSliderSpinBoxWidget" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkSliderWidget" name="pointSizeSliderSpinBoxWidget" native="true"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="pointGlyphTypeComboBox">
           <property name="accessibleDescription">
            <string>Glyph type combo box</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pointGlyphTypeLabel">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="ctkColorPickerButton" name="pointSelectedColorPickerButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Selected Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Lines</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="3">
        <widget class="QTableWidget" name="pointsTableWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>457</width>
           <height>0</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>4</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ctkColorPickerButton" name="lineUnselectedColorPickerButton">
         <property name="text">
          <string>Unselected Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ctkColorPickerButton" name="lineSelectedColorPickerButton">
         <property name="text">
          <string>Selected Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel_2">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="ctkSliderWidget" name="lineWidthSliderSpinBoxWidget_2" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="opacityLabel_3">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="ctkSliderWidget" name="lineOpacitySliderSpinBoxWidget_2" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="ambientLabel_3">
           <property name="text">
            <string>Ambient:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="ctkSliderWidget" name="lineAmbientSliderSpinBoxWidget_2" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="diffuseLabel_3">
           <property name="text">
            <string>Diffuse:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="ctkSliderWidget" name="lineDiffuseSliderSpinBoxWidget_2" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="specularLabel_3">
           <property name="text">
            <string>Specular:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="ctkSliderWidget" name="lineSpecularSliderSpinBoxWidget_2" native="true">
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Label Position:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="ctkSliderWidget" name="lineLabelPositionSliderSpinBoxWidget" native="true">
           <property name="toolTip">
            <string>Position the label along the line</string>
           </property>
           <property name="singleStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QCheckBox" name="lineLabelVisibilityCheckBox">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Label Visibility</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="toolTip">
            <string>Distance in mm between major ticks</string>
           </property>
           <property name="text">
            <string>Tick Spacing:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPlainTextEdit" name="lineTickSpacingPlainTextEdit">
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ROI">
      <attribute name="title">
       <string>ROI</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="qMRMLAnnotationROIWidget" name="ROIWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLAnnotationROIWidget</class>
   <extends>QWidget</extends>
   <header>GUI/qMRMLAnnotationROIWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerAnnotationModule.qrc"/>
  <include location="../qSlicerAnnotationModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qSlicerAnnotationModulePropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qSlicerAnnotationModulePropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
